packages/massa-proto-cli/src/ts-gen.ts

```diff
@@ -186,18 +187,45 @@ import {
   time, 
 } from "@massalabs/massa-web3";
 
+/**
+ * This interface represents the details of the transaction.
+ * 
+ * @see operationId - The operationId of the Smart Contract call
+ */
 export interface TransactionDetails {
   operationId: string;
 }
 
-export interface IEventPollerResult {
+/**
+ * This interface represents the result of the event poller.
+ * 
+ * @see isError - A boolean indicating wether the Smart Contract call has failed or not
+ * @see eventPoller - The eventPoller object
+ * @see events - The events emitted by the Smart Contract call
+ */
+export interface EventPollerResult {
   isError: boolean;
   eventPoller: EventPoller;
   events: IEvent[];
 }
 
-const MASSA_EXEC_ERROR = 'massa_execution_error';
+/**
+ * This interface is used to represents the outputs of the SC call.
+ * 
+ * @see outputs - The outputs of the SC call (optional)
+ * @see events - The events emitted by the SC call (optional)
+ * @see isError - A boolean indicating wether the SC call has failed or not
+ * @see error - The error message (optional)
```

Comment:
Why do you plan an isError flag and an error key? Why not simply using the error key ?